pkg/kn/commands/service_delete.go

```diff
@@ -53,7 +53,7 @@ func NewServiceDeleteCommand(p *KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "Deleted %s in %s namespace.\n", args[0], namespace)
+			fmt.Fprintf(cmd.OutOrStdout(), "Deleted '%s' in '%s' namespace.\n", args[0], namespace)
```

Comment:
Could we please add 'service' as extra context info (as this is easier to detect e.g. when grepping in logs when this output is redirected) ? Also, would prefer to have "namespace" before the actual namespace (as this is the usual order: <resource kind> <name> in message.

I.e. `Deleted service '%s' in namespace '%s'`

Whether to add punctuation or not should be follow a common convention over all messages (not sure how it is in the client).